README.adoc

```diff
@@ -97,7 +97,7 @@ To learn about the various Kubernetes resources that you can configure, see the
 
 == What you'll learn
 
-You will learn how to deploy two microservices in Open Liberty Docker containers to a local {kube} cluster.
```

Comment:
I think it might be a bit confusing to remove the word `Docker` in this sentence.

Reply:
I think removing the Docker sounds better.

Reply:
Plus, when we write Open Liberty beside Docker, it looks as though it's a trademarked name, which it is not. That was my initial thought when removing Docker from the sentence.